protocol/governance/contracts/modules/core/BaseElectionModule.sol

```diff
@@ -346,16 +346,32 @@ contract BaseElectionModule is
 
         uint newEpochIndex = store.currentElectionId + 1;
 
+        CrossChain.Data storage cc = CrossChain.load();
+        cc.broadcast(
+            cc.getSupportedNetworks(),
+            abi.encodeWithSelector(
+                this._recvResolve.selector,
+                election.winners.values(),
+                newEpochIndex
+            ),
+            _CROSSCHAIN_GAS_LIMIT
+        );
+    }
+
+    function _recvResolve(address[] calldata winners, uint256 newEpochIndex) external {
+        CrossChain.onlyOnChainAt(0);
```

Comment:
I just noticed this line here: `CrossChain.onlyOnChainAt(0);`

I think this should be removed since `_recvResolve` needs to be called on every network in order to distribute the NFTs on each chain.